cypress/features/regression/flatTabel.feature

```diff
@@ -0,0 +1,88 @@
+Feature: FlatTable component
+  I want to check Flat Table component properties
+
+  Background: Open FlatTable component default page
+    Given I open "Flattable" component page
+
+  @positive
+  Scenario: FlatTable cell has row header
+    When I check hasRowHeader checkbox
+    Then FlatTable has row header
+
+  @positive
+  Scenario: FlatTable has sticky header
+    When I check hasStickyHeader checkbox
+    Then FlatTable has sticky header
+
+  @positive
+  Scenario: Check that Flat Table has nine rows
+    # commented because of BDD default scenario Given - When - Then
+    # When I open "Flattable" component page
+    Then FlatTable has nine row
+
+  @positive
+  Scenario: Check that Flat Table has seven columns
+    # commented because of BDD default scenario Given - When - Then
+    # When I open "Flattable" component page
+    Then FlatTable has nine row
+
+  @positive
+  Scenario Outline: Check value in header cells
+  # commented because of BDD default scenario Given - When - Then
+  When I open "Flattable" component page
+  Then "<position>" header cell has value "<text>"
+  Examples:
+    | position | text              |
+    | first    | Client            |
+    | second   | Final Account Due |
+    | third    | Corp Tax Due      |
+    | fourth   | VAT due           |
+    | fifth    | Client Type       |
+    | sixth    | Categories        |
+    | seventh  | Products          |
+
+  @positive
+  Scenario Outline: Check value in <position> cells
+    # When I open "Flattable" component page
+    Then FlatTable <position> row contains proper inner context
+    Examples:
+      | position |
+      | 0        |
+      | 1        |
+      | 2        |
+      | 3        |
+      | 4        |
+      | 5        |
+      | 6        |
+
+  @positive
+  Scenario Outline: Check header and row of Flat Tabel is visible after scroll
+    Given I check hasStickyHeader checkbox
+      And I check hasRowHeader checkbox
+    When I scroll tabel content to right bottom
+    Then "<positionVisible>" header cell has value "<text>"
+      And FlatTable 3 row is visible
+      And FlatTable 4 row is visible
+      And FlatTable 5 row is visible
+      And FlatTable 6 row is visible
+    Examples:
+      | positionVisible | text        |
+      | first           | Client      |
+      | sixth           | Categories  |
+      | seventh         | Products    |
+
+  @positive
+  Scenario Outline: Check header of Flat Tabel is not visible after scroll
```

Comment:
the not visible row should be mentioned in `Scenario`

Reply:
fixed